tests/ctst/steps/azureArchive.ts

```diff
@@ -390,6 +391,37 @@ Then('object {string} should expire in {int} days', async function (this: Zenko,
     assert(diff >= days && diff < days + 0.1);
 });
 
+Given('that lifecycle is {string} for the {string} location',
+    async function (this: Zenko, status: string, locatiom: string) {
+        let path: string;
+        if (status === 'paused') {
+            path = `/_/lifecycle/pause/${locatiom}`;
+        } else {
+            path = `/_/lifecycle/resume/${locatiom}`;
+        }
+        await axios.post(`http://${this.parameters.backbeatApiHost}:${this.parameters.backbeatApiPort}${path}`);
+    });
+
+Then('the storage class of object {string} should be {string} for {int} seconds',
```

Comment:
Not sure if this function really does what you want it to do

"should be XXXX for YYYY seconds" implies that the storage classe remains the same during the time in parameter, but here if you sleep directly at the beginning of the function you just check that after the timeout it is of the right storage class...

Maybe you should checkk the storage class every second instead ?

Reply:
That said, I already implemented such a function in my PR so maybe it's worth waiting for mine to be merged..